README.md

```diff
@@ -18,14 +38,21 @@ import { MediapipeView } from "react-native-mediapipe";
 <MediapipeView color="tomato" />
 ```
 
+> Link to an example of this in use later.
+
 ## Contributing
 
 See the [contributing guide](CONTRIBUTING.md) to learn how to contribute to the repository and the development workflow.
 
+Join the [Community](https://discord.gg/ApuAzVnAaX) here! 
+<!--- (I want to turn this into a banner later down the line using the code on the top of the file.) --->
```

Comment:
I don't think a full-width banner is particularly advisable for a link to a discord invite. I see other readme.md's having a list of social media contact links (in our case, unclear whether we'll ever have more than discord), but the big banners are for company/product sites. 

In general, if there is future work to do, just file an "issue". You don't want to litter the checked-in stuff with things on what you want to do in the future (and you'll only ever find them again if you look through the code).

Reply:
I removed the text here for it, thanks for telling me it wasn't recommended